vendor_lib/imperas/imperas_DV_COREV/ChangeLog.md

```diff
@@ -1,6 +1,6 @@
 CV32E40P Change Log
```

Comment:
This changelog mentions the E40P, but I'm guessing it applies to all E40 cores.   You can fix that in a future PR if you like.